src/main/java/it/gov/pagopa/payhub/auth/config/SwaggerConfig.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.payhub.auth.config;
```

Comment:
move to a common package: probably this file could be copy/pasted in other future applications
```suggestion
package it.gov.pagopa.payhub.common.config;
```